File: http/src/main/java/cn/bluemobi/dylan/http/download/ProgressResponseListener.java
package cn.bluemobi.dylan.http.download;

/**
 * Created by YDL on 2017/6/8./**
 * 响应体进度回调接口，用于文件下载进度回调
 */

public interface ProgressResponseListener {
    void onResponseProgress(long bytesRead, long contentLength, boolean done);
}

File: http/src/main/java/cn/bluemobi/dylan/http/download/ProgressResponseListener.kt
package cn.bluemobi.dylan.http.download

/**
 * Created by YDL on 2017/6/8./ **
 * 响应体进度回调接口，用于文件下载进度回调
 */
interface ProgressResponseListener {
    fun onResponseProgress(bytesRead: Long, contentLength: Long, done: Boolean)
}